vocabularies/qld-mining-permits.ttl

```diff
@@ -0,0 +1,305 @@
+        minpermit:ExplorationPermitCoal,
+        minpermit:ExplorationPermitGeothermal,
+        minpermit:ExplorationPermitGreenhouseGas,
+        minpermit:ExplorationPermitMineral,
+        minpermit:ExplorationPermitSpecial,
+        minpermit:GeothermalLease,
+        minpermit:GreenhouseGasLease,
+        minpermit:MineralDevelopmentLicence,
+        minpermit:MiningClaim,
+        minpermit:MiningLease,
+        minpermit:PetroleumFacilityLicence,
+        minpermit:PetroleumLease,
+        minpermit:PetroleumPipelineLicence,
+        minpermit:PetroleumSurveyLicence,
+        minpermit:PotentialCommercialArea,
+        minpermit:PotentialGeothermalCommercialArea,
+        minpermit:PotentialGreenhouseGasStorageCommercialArea,
+        minpermit:ProspectingPermit,
+        minpermit:WaterMonitoringAuthority ;
+    skos:hiddenLabel "Tenement"@en ;
+    skos:prefLabel "Queensland Mining Permit"@en .
+
+minpermit:GeothermalAuthorities a skos:Collection ;
+    skos:altLabel "Geothermal permits"@en ;
+    skos:definition "An appropriate resource authority is required before you can conduct any exploration or production activity for geothermal energy in Queensland. These are issued under Queensland's Geothermal Energy Act 2010."@en ;
+    skos:hiddenLabel "Geothermal tenures"@en ;
+    skos:member minpermit:ExplorationPermitGeothermal,
+        minpermit:GeothermalLease,
+        minpermit:PotentialGeothermalCommercialArea ;
+    skos:prefLabel "Geothermal authorities"@en .
+
+minpermit:GreenhouseGasAuthorities a skos:Collection ;
+    skos:definition "An appropriate resource authority is required before you can conduct any exploration or production activity for greenhouse gas storage areas in Queensland. These are issued under Queensland's Greenhouse Gas Storage Act 2009."@en ;
+    skos:member minpermit:DataAcquisitionAuthority,
+        minpermit:ExplorationPermitGreenhouseGas,
+        minpermit:GreenhouseGasLease,
+        minpermit:PotentialGreenhouseGasStorageCommercialArea ;
+    skos:prefLabel "Greenhouse gas storage authorities"@en .
+
+minpermit:MiningAuthorities a skos:Collection ;
+    skos:altLabel "Mining permits"@en,
+        "Mining tenures"@en ;
+    skos:hiddenLabel "Mining tenements"@en ;
+    skos:member minpermit:ExplorationPermitCoal,
+        minpermit:ExplorationPermitMineral,
+        minpermit:ExplorationPermitSpecial,
+        minpermit:MineralDevelopmentLicence,
+        minpermit:MiningClaim,
+        minpermit:MiningLease,
+        minpermit:WaterMonitoringAuthority ;
+    skos:prefLabel "Mining authorities"@en .
+
+minpermit:PetroleumAndGasAuthority a skos:Collection ;
+    skos:altLabel "Petroleum and gas permit"@en ;
+    skos:definition """An appropriate resource authority is required before you can conduct any exploration or production activity for petroleum and gas in Queensland. These are issued under Queensland's Petroleum and Gas (Production and Safety) Act 2004. 
```

Comment:
triple quote marks at start and end of definition

Reply:
https://www.w3.org/TR/sparql11-query/#QSynLiterals

To facilitate writing literal values which themselves contain quotation marks or which are long and contain newline characters, SPARQL provides an additional quoting construct in which literals are enclosed in three single- or double-quotation marks.

The lines affected contain the newline character \n

Reply:
See above.